### PR TITLE
Rename config

### PR DESCRIPTION
**Pull request by wanghonglie**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
rename config

## Modification

rename config

### CODE REVIEW

File: configs/yolox/yolox_l_8x8-300e_coco.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+_base_ = './yolox_s_8x8-300e_coco.py'
```
Comment (ZwwWayne) [inline configs/yolox/yolox_l_8x8-300e_coco.py:1]: yolox_s_8xb8-300e_coco.py

File: configs/albu_example/mask-rcnn_r50-fpn_albu-1x_coco.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+_base_ = '../mask_rcnn/mask-rcnn_r50-fpn_1x_coco.py'
```
Comment (BIGWangYuDong) [inline configs/albu_example/mask-rcnn_r50-fpn_albu-1x_coco.py:1]: does mask_rcnn also need to rename to `mask-rcnn`

**Comment by BIGWangYuDong** [comment]
- README.md in each config folder also need to rename.
- docs also need to check whether should change the config name
- Fix Unit Test

File: configs/albu_example/mask-rcnn_r50-fpn_albu-1x_coco.py (hunk 2 above)
Comment (wanghonglie) [reply configs/albu_example/mask-rcnn_r50-fpn_albu-1x_coco.py:1]: Folders do not need to be renamed.

**Comment by ZwwWayne** [comment]
continued in https://github.com/open-mmlab/mmdetection/pull/8603

**Pull request closed without merging**